internal/impl/config.go

```diff
@@ -40,6 +40,13 @@ type Config struct {
 
 	// Nixpkgs specifies the repository to pull packages from
 	Nixpkgs NixpkgsConfig `json:"nixpkgs,omitempty"`
+
+	// Reserved to allow including other config files. Proposed format is:
+	// path: for local files
+	// https:// for remote files
+	// plugin: for built-in plugins
+	// This is similar to nix inputs
```

Comment:
Mind elaborating on what you mean by "similar to nix inputs": in what way?

Reply:
nix flake inputs are urls of this format (path, https, github, etc) so we are using the same format.

Reply:
I see, got it. Perhaps "This **format** is similar to nix inputs"